bonfire/Language/en/Bonfire.php

```diff
@@ -14,4 +14,11 @@
 
     'notAuthorized' => 'You do not have the correct permissions.',
     'unknownAction' => 'Did not recognize the action you requested.',
+
+    'nested' => [
+        'fileSystem' => [
+            'lastModified' => 'Last Modified',
```

Comment:
No need to make these this specific. They're pretty generic phrases that can potentially be reused elsewhere. Please make them top-level standalone entries like the rest of them.

Reply:
Done.